### PR TITLE
update docs to reflect alarm cron 5 fields by default

### LINKED ISSUE #1545: Update docs for alarm to use 5 fields cron syntax by default
docs examples should show 5 fields, like typical/standard cron syntax

Users are getting confuse with 6 fields by having the first field be seconds and not minutes

the cron variable still supports 6 fields, but it should be noted as an custom syntax for users that might want to specify sub-minute granularity.

cc @bjustin-ibm

### PR DESCRIPTION
**Pull request by csantanapr**
Fixes #1545

### CODE REVIEW

**Comment by csantanapr** [comment]
@rabbah @bjustin-ibm here is draft please comment on how to adjust.

File: docs/catalog.md [hunk 1]
```diff
@@ -221,25 +221,30 @@ The package includes the following feed.
...
+The following is an example of creating a trigger that will be fired once every 2 minutes with `name` and `place` values in the trigger event.
 
   ```
-  $ wsk trigger create periodic --feed /whisk.system/alarms/alarm --param cron "*/20 * * * * *" --param trigger_payload "{\"name\":\"Odin\",\"place\":\"Asgard\"}"
+  $ wsk trigger create periodic --feed /whisk.system/alarms/alarm --param cron "*/2 * * * *" --param trigger_payload "{\"name\":\"Odin\",\"place\":\"Asgard\"}"
   ```
 
 Each generated event will include as parameters the properties specified in the `trigger_payload` value. In this case, each trigger event will have parameters `name=Odin` and `place=Asgard`.
 
+**Note**: The parameter `cron` also supports a custom syntax of six fields, were the first field represents seconds.
```
Comment (bjustin-ibm) [inline docs/catalog.md]: were -> where

File: docs/catalog.md [hunk 2]
```diff
@@ -221,25 +221,30 @@ The package includes the following feed.
...
-  $ wsk trigger create periodic --feed /whisk.system/alarms/alarm --param cron "*/20 * * * * *" --param trigger_payload "{\"name\":\"Odin\",\"place\":\"Asgard\"}"
+  $ wsk trigger create periodic --feed /whisk.system/alarms/alarm --param cron "*/2 * * * *" --param trigger_payload "{\"name\":\"Odin\",\"place\":\"Asgard\"}"
   ```
 
 Each generated event will include as parameters the properties specified in the `trigger_payload` value. In this case, each trigger event will have parameters `name=Odin` and `place=Asgard`.
 
+**Note**: The parameter `cron` also supports a custom syntax of six fields, were the first field represents seconds. 
+For more details about using this custom cron syntax, see: https://github.com/ncb000gt/node-cron. 
+Here is an example using six fields notation:
+  - `*/30 * * * * *`: every thirty seconds.
```
Comment (bjustin-ibm) [inline docs/catalog.md]: I kind of like having it at the very end. This makes it more along the lines of, "Oh by the way, if you happen to need this, here it is" which seems appropriate to me.

File: docs/catalog.md (hunk 1 above)
Comment (bjustin-ibm) [inline docs/catalog.md]: were -> where

Comment (bjustin-ibm) [inline docs/catalog.md]: LGTM apart from this one correction.

Comment (csantanapr) [inline docs/catalog.md]: I will correct

**Comment by csantanapr** [comment]
@bjustin-ibm I fixed here: https://github.com/openwhisk/openwhisk/pull/1551/commits/1a4c45fc367964f9b3112fd1945a6f17e1c794c2

squash and merge if there no more comments.

**Comment by bjustin-ibm** [comment]
Squashed and merged as dbbdc3b

**Pull request closed without merging**